src/main/java/lotto/util/LottoNumberParser.java

```diff
@@ -0,0 +1,78 @@
+package lotto.util;
+
+import lotto.exception.ErrorMessage;
+import lotto.exception.LottoException;
+import lotto.model.constant.LottoRules;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * 로또 번호 입력을 파싱하는 유틸리티 클래스
+ */
+public final class LottoNumberParser {
+    public static final String NUMBER_DELIMITER = ",";
+
+    private LottoNumberParser() {
+    }
+
+    /**
+     * 쉼표로 구분된 로또 번호 문자열을 파싱하여 숫자 목록으로 반환
+     *
+     * @param input 쉼표로 구분된 번호 문자열
+     * @return 파싱된 번호 목록
+     * @throws LottoException 입력이 유효하지 않은 경우
+     */
+    public static List<Integer> parseNumbers(String input) {
+        try {
+            String[] parts = input.trim().split(NUMBER_DELIMITER);
+            validateEmptyParts(parts);
+            return Arrays.stream(parts)
+                    .map(String::trim)
+                    .map(Integer::parseInt)
+                    .collect(Collectors.toList());
+        } catch (NumberFormatException e) {
+            throw new LottoException(ErrorMessage.WINNING_PARSE);
+        }
+    }
+
+    /**
+     * 빈 문자열 부분이 있는지 검증
+     *
+     * @param parts 분할된 문자열 배열
+     * @throws LottoException 빈 부분이 있을 경우
+     */
+    private static void validateEmptyParts(String[] parts) {
+        for (String part : parts) {
+            if (part.trim().isEmpty()) {
+                throw new LottoException(ErrorMessage.WINNING_EMPTY);
+            }
+        }
+    }
```

Comment:
[마이너] 생각해보니까 저는 이 예외 처리가 없었네요,, 소미님 코드 보고 깨닫고 갑니다

[코드] 음... 메서드를 분리하거나 아래와 같이 이런식으로 눈속임? 하는 방식은 어떨까요 ㅋㅋ
근데 아마 우테코 스타일 상 적용하면 return이 {} 블록으로 감싸지도록 수정될 것 같긴한데
```suggestion
    /**
     * 빈 문자열 부분이 있는지 검증
     *
     * @param parts 분할된 문자열 배열
     * @throws LottoException 빈 부분이 있을 경우
     */
    private static void validateEmptyParts(String[] parts) {
        for (String part : parts) {
            if (!part.trim().isEmpty()) return;
            throw new LottoException(ErrorMessage.WINNING_EMPTY);
        }
    }
```

Reply:
오!! 이런 식으로도 해볼 수도 있군요 👍🏻

Reply:
오!! 이런 식으로도 해볼 수도 있군요 👍🏻